Moeification.sh

```diff
@@ -273,6 +275,20 @@ if [ ! $? -eq 0 ]; then
 else
     echo "$(eval_gettext "Downloaded the Your Name 2 wallpaper.")"
 fi
+wget https://i.redd.it/02c3t2v8zl561.png -O "/usr/share/weebpapers/(Moe) Yorimoi.png" > /dev/null 2>&1
+if [ ! $? -eq 0 ]; then
+    echo "$(eval_gettext "Oh no, an error occured downloading the Yoimoi wallpaper, the girls will never reach antarctica!") $(eval_gettext "Aborting now.")"
```

Comment:
Just FYI I'd do this myself post-merge, but... I'd personally change 'antarctica' to 'Antarctica', and change it from 'wallpaper, the' to 'wallpaper. The'.